### ISSUE TITLE
Archive on first add doesn't seem to work

### ISSUE THREAD

**Issue by patridge** [omnivore-app/omnivore#2751]
I like to keep a common archive of things I have read, and not just "want to read". So, I like to add to Omnivore and immediately archive things I just finished reading.

Unfortunately, that process doesn't seem to work as I expect it.

## Expected

If I archive an article, it should be only in the Archived or All lists, even if I just added it to Omnivore a second before archiving it.

## Actual

Any content I add and then immediately archive stays at the top of my Inbox and Read Later lists. It doesn't appear that the Archive process does anything. Any articles I need to archive requires going to the inbox list and archiving them again for there.

## Environment

I haven't tried this everywhere I can, but this definitely happens when adding and archiving immediately on iOS from the Share pop-up and Chrome from the extension pop-up.

iOS app v1.28.0 and whatever Chrome extension is latest today.

**Comment by jacksonh** [comment]
I'm not able to replicate this one. Here's what I am seeing: https://www.loom.com/share/10a51df25aac43ae828812c1d7413675

**Comment by patridge** [comment]
Maybe this is just a disconnect from my expectations of the default saved search list, Inbox.

I was adding the item and immediately archiving, similar to how you did it, but then going to Omnivore from the first extension option to see my saved content to read next.

At that point, the archived article is at the very top of my inbox, which I equated to my experience from other platforms as "unread inbox". So, I wasn't expecting to see an article there when I archived it. (At that point, I'm done caring about that article at all, until any time in the future when I need to search for "that one article I read about X".)

@jacksonh Is there an equivalent to "Unreads" that I can make my default view in Omnivore? Or maybe I'm misunderstanding the Continue Reading section. If I haven't started reading something yet on Omnivore, would I expect it to still show there? I was expecting that to be things I started reading after adding them to Omnivore, not net-new content I just added. I regularly add articles via iOS link sharing without even giving them the first glance.

For the immediate bug concern about it not archiving, I guess we are good to go. I'll close this.

Looking in the [Archived list](https://omnivore.app/home?q=in%3Aarchive), I do see new articles showing there as expected. Additionally, I confirmed the addition to the Archived list through iOS as well.

**Comment by jacksonh** [comment]
Hey @patridge we can probably start to build what you want using Saved Searches now. I think we might have an issue with the web version of Continue Reading where it is showing items that haven't been started yet.

**Comment by patridge** [comment]
@jacksonh I just noticed on iOS that there is a Read Later list, which appears to be what I was looking for. I don't see an equivalent list on my web interface, though.

<img width="164" alt="image" src="https://github.com/omnivore-app/omnivore/assets/713665/da222bac-9ed4-472b-a7ea-38949c272b85">

Unfortunately, I have also reproduced the original not-marked-as-archived issue on iOS a bunch more times now. It might be some sort of race between sharing an article from Chrome to Omnivore and the archive request (**More Actions** and **Archive** from the resulting pop-up). If things haven't settled on the share to Omnivore, the archiving doesn't seem to take at all, and the article is still in Read Later but not in Archived.

**Comment by jacksonh** [comment]
Yeah the iOS `Continue Reading` section is probably what you want, but it can do some extra filtering on the client side that the web can't yet. We just need to fix this Continue Reading query a bit on web.

**Comment by Podginator** [comment]
Closing due to age, and the self-hosted nature of Omnivore. Done as a batch action. If this issue still exists, please reopen it.
